File: Actividad 2 Suma y Promedio de un Arreglo/Actividad2.java
import java.util.Scanner;

public class Actividad2 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int[] calificaciones = new int[10];
        int suma = 0;

        // Ingreso de datos
        for (int i = 0; i < calificaciones.length; i++) {
            System.out.print("Ingrese la calificación " + (i + 1) + ": ");
            calificaciones[i] = sc.nextInt();
        }

        // Suma total
        for (int nota : calificaciones) {
            suma += nota;
        }

        // Promedio
        double promedio = (double) suma / calificaciones.length;

        // Contar mayores al promedio
        int mayores = 0;

        for (int nota : calificaciones) {
            if (nota > promedio) {
                mayores++;
            }
        }

        // Resultados
        System.out.println("\nResultados:");
        System.out.println("Suma total: " + suma);
        System.out.println("Promedio: " + promedio);
        System.out.println("Cantidad de notas por encima del promedio: " + mayores);

        sc.close();
    }
}

File: Actividad 2 Suma y Promedio de un Arreglo/actividad2.py
calificaciones = []
suma = 0

# Ingreso de datos
for i in range(10):
    nota = int(input(f"Ingrese la calificación {i + 1}: "))
    calificaciones.append(nota)

# Suma total
for nota in calificaciones:
    suma += nota

# Promedio
promedio = suma / len(calificaciones)

# Contar notas mayores al promedio
mayores = 0

for nota in calificaciones:
    if nota > promedio:
        mayores += 1

# Resultados
print("\nResultados:")
print("Suma total:", suma)
print("Promedio:", promedio)
print("Cantidad de notas por encima del promedio:", mayores)
